project.properties

```diff
@@ -1,12 +1,7 @@
 forge_version=10.13.4.1558
 minecraft_version=1.7.10
-snapshot=20140925
 group_id=teamjm
 mod_id=journeymap
-jm_major=5
-jm_minor=1
-jm_micro=4
-jm_patch=p8
 fairPlay=FairPlay
 unlimited=Unlimited
 noRadar=NoRadar
```

Comment:
As long as you check out the repository with git tags, it will keep the version when building locally - you can always check with `./gradlew printVersion` (currently it's `5.1.4p5-rfg-buildscript.22+c5c1d4797c` for me on this branch, looks like you didn't push git tags for p6-p8 builds).

You can also easily override it by environment variables like so: `VERSION=5.1.4p8 ./gradlew build` (linux) or `SET VERSION=5.1.4p8 && .\gradlew.bat build` (windows).

If you prefer the properties file versioning mechanism, I can disable the git-based versioning and use that instead.

Reply:
Yea, I did not create tags for those because I was building and publishing locally. I had no workflows created for this project. Mainly because I wanted to wait until I could get Modrinth publishing working first. Since I have to manually upload currently. This is fine. 
Thanks!

Reply:
> As long as you check out the repository with git tags, it will keep the version when building locally - you can always check with `./gradlew printVersion` (currently it's `5.1.4p5-rfg-buildscript.22+c5c1d4797c` for me on this branch, looks like you didn't push git tags for p6-p8 builds).
> 
> You can also easily override it by environment variables like so: `VERSION=5.1.4p8 ./gradlew build` (linux) or `SET VERSION=5.1.4p8 && .\gradlew.bat build` (windows).
> 
> If you prefer the properties file versioning mechanism, I can disable the git-based versioning and use that instead.

can you add this information to the readme please?